RevenueCatUI/Helpers/Constants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constants.swift
+//  
+//
+//  Created by Nacho Soto on 7/13/23.
+//
+
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+enum Constants {
+
+    static let defaultAnimation: Animation = .easeIn(duration: 0.2)
```

Comment:
Just to start polishing things up a bit, but we'll obviously revisit all this stuff.